src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+        Lottery[] balls = {ball1, ball2, ball3};
+        for (Lottery ball : balls) {
+            System.out.println(ball.getRandomBall());
+        }
```

Comment:
A few points about this part:
1) Variables names should reflect their content. You have three "ball" variables, that contain lottery objects, this doesn't seem right.
2) You need only 1 lottery object, to create any number of balls, because the lottery class has a particular method for this, which makes a new ball every time you call that method (getRandomBall())